### PR TITLE
feat: CHK-3498 remove recaptcha post sessions

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Limit reCaptcha request removing it from api that doesn't expose sensitive data like POST `/sessions`

### List of changes

- remove recaptcha for POST `/sessions`
- rename policy file

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-app/api/ecommerce-checkout/v1/_post_sessions.xml.tpl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<policies>
+    <inbound>
+    <rate-limit-by-key 
+    calls="150"
```
Comment (CianoDanilo) [inline src/domains/ecommerce-app/api/ecommerce-checkout/v1/_post_sessions.xml.tpl:4]: Isn't the threshold of 150 calls per 10 seconds too high?

Comment (ciuffagianluca) [reply src/domains/ecommerce-app/api/ecommerce-checkout/v1/_post_sessions.xml.tpl:4]: We set this threshold together with @infantesimone reasoning about 2 points:

1. this is the same rate limit set in the base policy for payment-request verify
2. thinking about one office with a proxy to go out in internet that expose same ip for all machines

Considering these 2 points we decide to set this value for this threshold

**Review by EmanueleBVtech — DISMISSED** [review]

**Review by CianoDanilo — APPROVED** [review]

**Review by pietro-tota — APPROVED** [review]
🚀 well done!

File: src/domains/ecommerce-app/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!-- markdownlint-disable -->
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+<!-- BEGIN_TF_DOCS -->
```
Comment (pietro-tota) [inline src/domains/ecommerce-app/README.md:2]: maybe here we have to rollback those pre-commit changes?

Comment (ciuffagianluca) [reply src/domains/ecommerce-app/README.md:2]: Why should we rollback?

**Pull request merged**